src/Symfony/Component/HttpKernel/Tests/Controller/LegacyArgumentResolverTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Controller;
+
+use Symfony\Component\HttpKernel\Controller\LegacyArgumentResolver;
+use Symfony\Component\HttpKernel\Controller\ControllerResolver;
+use Symfony\Component\HttpKernel\Tests\Fixtures\Controller\VariadicController;
+use Symfony\Component\HttpFoundation\Request;
+
+class LegacyArgumentResolverTest extends \PHPUnit_Framework_TestCase
+{
+    public function testGetArguments()
+    {
+        $resolver = new LegacyArgumentResolver();
+
+        $request = Request::create('/');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\Symfony\Component\HttpFoundation\Request' in method 'testGetArguments'.](https://www.codacy.com/app/git_9/symfony/file/2331529211/issues/source?bid=3207821&fileBranchId=3207847#l25)